src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilHearingJudicialRoleMappingTest.java

```diff
@@ -209,11 +216,20 @@ void shouldTakeJudicialAccessProfileConvertToJudicialOfficeHolderThenReturnRoleA
                     assertEquals("London", r.getAttributes().get("primaryLocation").asText());
                 }
                 if (judgeRoleNamesWithWorkTypes.contains(r.getRoleName())) {
-                    assertEquals("hearing_work,decision_making_work,applications",
+                    assertEquals("decision_making_work,applications",
+                            r.getAttributes().get("workTypes").asText());
+                } else if (judgeRoleNamesWithExtendedWorkTypes.contains(r.getRoleName())) {
+                    assertEquals("decision_making_work,applications,Multi_Track_decision_making_work,"
+                                   + "Intermediate_Track_decision_making_work",
                             r.getAttributes().get("workTypes").asText());
                 } else if (r.getRoleName().contains("leadership-judge")) {
                     assertEquals("LDN", r.getAttributes().get("region").asText());
-                    assertEquals("access_requests",
+                    assertEquals("decision_making_work,applications,access_requests,"
+                            + "Multi_Track_decision_making_work,Intermediate_Track_decision_making_work",
+                            r.getAttributes().get("workTypes").asText());
+                } else if (r.getRoleName().contains("task-supervisor")) {
+                    assertEquals("LDN", r.getAttributes().get("region").asText());
+                    assertEquals("hearing_work,decision_making_work,applications",
```

Comment:
The current confluence page says: `decision_making_work, applications`.  Can you confirm if change required.

Reply:
Resolved in 257da117577e0d77ecdb997b40906c99ba8c2e87.